docs/examples/amor.ipynb

```diff
@@ -33,9 +33,9 @@
     "    **default_parameters,\n",
     "    QBins: sc.geomspace(dim='Q', start=0.008, stop=0.075, num=200, unit='1/angstrom'),\n",
     "    SampleRotation[Sample]: sc.scalar(0.7989, unit='deg'),\n",
-    "    Filename[Sample]: \"sample.nxs\",\n",
+    "    PoochFilename[Sample]: \"sample.nxs\",\n",
     "    SampleRotation[Reference]: sc.scalar(0.8389, unit='deg'),\n",
-    "    Filename[Reference]: \"reference.nxs\",\n",
+    "    PoochFilename[Reference]: \"reference.nxs\",\n",
     "}"
    ]
   },
```

Comment:
Should we also have a short cell that explains how to load local file by setting ``Filename[Run]`` in the params here, maybe...?